### PR TITLE
Add post_build.sh to .gitignore

### PR DESCRIPTION
**Pull request by MysteriousLog6**
Add post_build.sh to .gitignore

### CODE REVIEW

**Comment by JF002** [comment]
I might be wrong, but I don't think this is needed:  `post_build.sh` is generated by cmake in the build folder, which should not be tracked by git.
In which situation did you notice that git would add this file?

**Comment by Avamander** [comment]
> In which situation did you notice that git would add this file?

When you don't follow best-practice and run `cmake` in the project's root folder (instead of `build`). It doesn't get instantly added, but people have accidentally added it to git themselves.

File: src/main.cpp [hunk 1]
```diff
@@ -125,7 +125,6 @@ Pinetime::Controllers::FS fs {spiNorFlash};
 Pinetime::Controllers::Settings settingsController {fs};
 Pinetime::Controllers::MotorController motorController {settingsController};
 
-
```
Comment (Avamander) [inline src/main.cpp:128]: This change is not needed

**Review by Avamander — CHANGES_REQUESTED** [review]
This doesn't hurt and can avoid someone accidentally adding these files, even if they aren't running `cmake` in the right folder. If that one line is reverted it's fine to merge.

**Comment by Avamander** [comment]
I'll close this as it's a counter to cmake misuse, which might have future unintended consequences we don't want to deal with.

**Pull request closed without merging**